data-pipeline/src/trace_exporter.rs

```diff
@@ -117,6 +132,9 @@ pub struct TraceExporter {
     // TODO - do something with the response callback - https://datadoghq.atlassian.net/browse/APMSP-1019
     _response_callback: Option<Box<dyn ResponseCallback>>,
     runtime: Runtime,
+    // None if dogstatsd is disabled
+    dogstatsd: Option<Flusher>,
```

Comment:
IIUC, this code isn't yet called and when we'll call it, we'll pass an flusher if the tracer have health metrics enabled. Is that right?

Reply:
Yep! By default there's no dogstatsd flusher, but when building the trace exporter if one is provided it enables health metrics